build.sbt

```diff
@@ -166,6 +166,12 @@ lazy val services = project
   .dependsOn(cwlV1_0LanguageFactory % "test->test")
   .dependsOn(core % "test->test")
 
+lazy val hybridCarboniteMetadataService = project
```

Comment:
This is an artifact of "this isn't real yet", right? i.e. why it's not just plugged in?

Reply:
and/or dodging issues around the `IOActor` currently being in `engine`? 🙂